http/server_test.ts

```diff
@@ -127,7 +127,7 @@ class MockListener implements Deno.Listener {
 }
 
 Deno.test(
-  "Server.addrs should expose the addresses the server is listening on",
+  "Server() exposes the addresses the server is listening on as addrs property",
```

Comment:
Can you please remove the round brackets on class names?

Reply:
done